File: CS/DocumentExportWithTokenBasedAuthentication/Startup.cs
using System;
using System.Collections.Generic;
using System.Linq;
using Microsoft.Owin;
using Owin;

[assembly: OwinStartup(typeof(T680906.Startup))]

namespace T680906
{
    public partial class Startup
    {
        public void Configuration(IAppBuilder app)
        {
            ConfigureAuth(app);
        }
    }
}

File: VB/DocumentExportWithTokenBasedAuthentication/Startup.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports Microsoft.Owin
Imports Owin

<Assembly: OwinStartup(GetType(T680906.Startup))>

Namespace T680906
	Partial Public Class Startup
		Public Sub Configuration(ByVal app As IAppBuilder)
			ConfigureAuth(app)
		End Sub
	End Class
End Namespace
